ansible/build-vars.yml

```diff
@@ -0,0 +1,9 @@
+nodejs_app_name: universal
+nodejs_app_install_dir: /opt/{{ nodejs_app_name }}
```

Comment:
@avtar @gtirloni for the current universal container, the install directory is `/opt/universal` (confirmed by running both the Ansible-built and the previous container side by side, and `-it`'ing into each) and `npm test` fails on both due to the inability to resolve universal.

Since it looks like all the downstream containers are actually being built `FROM kaspermarkus/universal` rather than `gpii/universal` it would probably be safe to change this to run the tests during the container build process. I'll experiment.